src/applications/model/gaming-streaming-server.cc

```diff
@@ -249,9 +248,13 @@ void
 GamingStreamingServer::ScheduleNextTx (int idx)
 {
   NS_LOG_FUNCTION (this << idx);
-  Time nextTx = MilliSeconds (m_trafficStreams[idx].m_interArrivalTimesVariable->GetInteger ());
-  m_trafficStreams[idx].m_sendEvent = Simulator::Schedule (nextTx,
-                                          &GamingStreamingServer::Send, this, idx);
+  double nextTx = -1;
```

Comment:
```suggestion
  Time nextTx = Seconds (-1);
  do
      {
         nextTx = MilliSeconds (m_trafficStreams[idx].interArrivalTimesVariable->GetValue ());
      } while (nextTx.IsStrictlyNegative ());
  m_trafficStreams[idx].sendEvent = Simulator::Schedule (nextTx,
                                                         &GamingStreamingServer::Send, this, idx);   
```

Reply:
https://www.nsnam.org/doxygen/classns3_1_1_time.html#abbca8efb0d4943ac1514f78c1f770535

with this while loop we would need one less comparison